examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -151,9 +92,41 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
     );
   }
 
+  Widget _buildCountdownTimerPicker(BuildContext context) {
+    return new GestureDetector(
+      onTap: () async {
```

Comment:
you're not doing anything with the result here, you can just skip this async and the next await I think